cvts/_data_retrieval.py

```diff
@@ -31,19 +29,22 @@ def vehicle_trace(vehicle_id:int, from_instant:int, to_instant:int)->dict:
                 'type':    'via'
             }
     """
-    dates = [datetime.fromtimestamp(epochtime) for epochtime in range(from_instant, to_instant + DAY, DAY)]
     data = []
     for date in dates:
-        fldr = join(DATA_LAKE, str(date.year), str(date.month).zfill(2), datetime.strftime(date, "%Y%m%d"))
+        fldr = join(
+            RAW_PATH,
+            str(date.month).zfill(2),
+            date.strftime("%Y%m%d"))
         if not isdir(fldr):
             continue
-        file = join(fldr, f'vehicle_{vehicle_id}.gzip')
-        if not isfile(file):
+        filename = join(fldr, '{}.gzip'.format(vehicle_id))
```

Comment:
The files are called vehicle_XXXXX.gzip, so this shouldn't work

Reply:
It does, because I treat the filename as the vehicle id... it was too much effort to try and figure out mapping between names and ids.